docs/tutorials/coverage.md

```diff
@@ -0,0 +1,164 @@
+# Coverage
+
+haskell.nix can generate coverage information for your package or
+project using Cabal's inbuilt hpc support.
+
+## Pre-requisites
+
+It is currently required that you enable coverage for each library you
+want coverage for prior to attempting to generate a coverage report. I
+hope to fix this before merging this PR:
+
+```nix
+  haskell-nix.cabalProject ({
+    src = pkgs.haskell-nix.haskellLib.cleanGit {
+      name = "haskell-nix-project";
+      src = ./.;
+    };
+    modules = [
+      {
+        packages.package-1.components.library.doCoverage = true;
+        packages.package-2.components.library.doCoverage = true;
+      }
+    ];
+  });
+```
+
+## Per-package
+
+```bash
+nix-build default.nix -A $pkg.coverageReport
+```
+
+This will generate a coverage report for the package you requested.
+The directory tree will look something like this:
```

Comment:
I think it would be fine to refer to your other document for this.

Reply:
Addressed https://github.com/input-output-hk/haskell.nix/pull/762/commits/0476ecc40d6bc973f51b6d0ce4bc84bc9fb43901.